### DISCUSSION TITLE
Slash commands rate limit

### DISCUSSION THREAD

**Discussion by jdadone** [discordjs/discord.js#7059] [Q&A] [answered]
Hello there! Is there any way to limit the rate of slash commands per user the way it's done for regular messages when slowmode is active? We've implemented slowmode logic on our side and the bot reply accordingly, but it would be awesome if the input was prevented and the cooldown timer shown.

Thanks for your time!

**Comment by manuelvleeuwen** [comment, marked as answer]
That is not a feature Discord natively supports no
